packages/@okta/vuepress-site/docs/guides/configure-signon-policy/before-you-begin/index.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Before you begin
+---
+
+## What is a sign-on policy
+
+An Okta Sign-On Policy helps you control who can sign in and how a user is allowed to sign in to Okta, including whether they are challenged for MFA and how long they are allowed to remain signed in before re-authenticating. For example, add a rule to the Okta Sign-On Policy when you need to make sure that only users who are inside your [corporate network](/docs/reference/api/policy/#network-condition-object) can access your application, or you need to exclude certain roles in your organization from accessing it.
+
+You can specify any number of policies and the order in which they are executed. If a policy in the list doesn't apply to the user trying to sign in, the system moves to the next policy. There is one required policy named **Default**. By definition, the default policy applies to all users.
+
+In addition to the default policy, which you can't delete, there may be another policy named **Legacy** that is present only if you have already configured MFA. This policy reflects the MFA settings that were in place when you enabled your sign-on policy and ensures that no changes in MFA behavior occur unless you modify your policy. If needed, you can delete it.
+
+> **Note:** See [Policies](/docs/concepts/policies) for an overview of the supported Okta policies and how they work.
+
+In addition to the Okta Sign-On Policy, there is a sign-on policy for each application that determines the extra levels of authentication that you may want performed before an application can be accessed. Add a rule, for example, to prompt groups that are assigned to your app to re-authenticate after 60 minutes.
```

Comment:
People tend to get tripped up by this and I think it would be worth acknowledging that in this guide. Something to the effect of:

> *Important* Keep in mind that in addition to the organization wide policies described here, you can also configure a sign-on policy for each ...